### PR TITLE
Fix CI breakage w/ 22.04

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

File: bootstrap
Comment (kamoltat) [inline bootstrap:1]: I'm guessing this fixes: `Unable to fetch some archives` error we have been seeing.

**Review by kamoltat — APPROVED** [review]
LGTM

**Comment by kamoltat** [comment]
Note: I think once we merge, it will bootstrap the new main and dependency CI will be back in business.

File: bootstrap
Comment (zmc) [reply bootstrap:1]: yeah, it seems like the issue is GitHub's 22.04 image carries out-of-date apt repo data, and what fixes it is running `apt-get update`

**Pull request merged**